### ISSUE TITLE
Sharing a direct link to a picture

### ISSUE THREAD

**Issue by Simon-Boyer** [photoview/photoview#745]
**Is your feature request related to a problem? Please describe.**
Sometimes I want to use direct links to picture. For example when I am publishing a blog post, I want to be able to share the picture and point to it directly. Right now the sharable link point to the picture inside the photoview UI.

**Describe the solution you'd like**
We could add, on the public link list, a second "copy link" button that copies the direct link instead of the normal link.

**Describe alternatives you've considered**
It is possible to get the direct link by opening the shared picture, right-clicking on it and doing a "Copy image link", but that's not really an ideal flow for that.

**Additional context**
I don't think it's too big of a change, if you're interested in that feature I could open a PR

**Comment by viktorstrate** [comment]
That would be a nice addition. A PR would be very welcome :)

**Comment by noel123007** [comment]
What happens if the share is password protected? Can the  "copy direct link" be hidden? @viktorstrate

**Comment by kkovaletp** [comment]
> What happens if the share is password protected?

Yes, it looks logical not to provide direct links for password-protected media. They fail with error 403 anyway.
